utils/gyb.py

```diff
@@ -304,7 +309,7 @@ def splitGybLines(sourceLines):
     dedents = 0
     try:
         for tokenKind, tokenText, tokenStart, (tokenEndLine, tokenEndCol), lineText \
-            in tokenize.generate_tokens(sourceLines.__iter__().next):
+            in tokenize.generate_tokens(partial(next, sourceLines.__iter__())):
```

Comment:
`lambda` is a keyword to create an anonymous function in Python. Here's a short demo:

```python
>>> some_string = "hello!"
>>> some_string
'hello!'
>>> some_function = lambda: some_string
>>> some_function()
'hello!'
>>> another_function = lambda s: some_string + s
>>> another_function(" world!")
'hello! world!'
```

So in this particular case, you'd probably want to do something like this:

```python
tokenize.generate_tokens(lambda: next(sourceLinesIter))
```

which results in a callable that returns